background.js

```diff
@@ -30,3 +30,7 @@ chrome.app.runtime.onLaunched.addListener(function() {
     win.onClosed.addListener(closeSerialPorts);
   });
 });
+
+chrome.runtime.onInstalled.addListener(function(evt){
+  chrome.storage.local.set({newVersion: 'newVersion'}, function() {});
```

Comment:
can a boolean be used here instead of a string as the value for the `newVersion` property?  Does the callback have to be given?